tests/tasks/setup_test_data.yml

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Get LSR_RHC_TEST_DATA environment variable
+  set_fact:
+    lsr_rhc_test_data_file: "{{ lookup('env', 'LSR_RHC_TEST_DATA') }}"
+  delegate_to: 127.0.0.1
+  become: false
+
+- name: Import test data
+  include_vars:
+    file: "{{ lsr_rhc_test_data_file }}"
+  when: lsr_rhc_test_data_file | length > 0
+  delegate_to: 127.0.0.1
```

Comment:
I don't think you need to delegate here - include_vars should work against the control node